spec/project/project_requires_spec.rb

```diff
@@ -1,8 +1,19 @@
 RSpec.describe 'Project requires' do
+  # rubocop:disable RSpec/MultipleExpectations
   it 'alphabetizes cop requires' do
-    source   = SpecHelper::ROOT.join('lib', 'rubocop-rspec.rb').read
-    requires = source.split("\n").grep(%r{rubocop/cop/rspec/[^(?:cop)]})
+    source   = SpecHelper::ROOT.join('lib', 'rubocop', 'cop', 'rspec_cops.rb')
+    captures = source.read.scan(%r{^(require_relative 'rspec/(.*?/)?(.*?)')$})
 
-    expect(requires.join("\n")).to eql(requires.sort.join("\n"))
+    # Sanity check in case these move again.
+    expect(captures).not_to be_empty
+
+    require_statements = captures.map(&:first).join("\n")
+    sorted_require_statements =
+      captures.sort_by do |_require_statment, cop_category, name|
+        [cop_category || 'rspec', name]
+      end.map(&:first).join("\n")
```

Comment:
Do we need the `.join("\n")`s? Can’t we just compare the arrays?

Reply:
I was wondering that at first too, but it produces a much nicer diff:
![image](https://user-images.githubusercontent.com/4211073/41407320-e6a3836a-6f83-11e8-94cc-1e5569a7d959.png)

v.s.

![image](https://user-images.githubusercontent.com/4211073/41407408-2cb87fd6-6f84-11e8-9b99-465ceb51a4e9.png)